db/seeders/letter_template_seeder.rb

```diff
@@ -1,17 +1,20 @@
 class LetterTemplateSeeder
   #rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
   def seed!
-    puts "----Seeding Letter Templates----"
+    puts "---- Seeding Letter Templates ----"
 
     rec = LetterTemplate.find_by(abbreviation: 'prisoner-acknowledgement')
     rec = LetterTemplate.new if rec.nil?
     rec.update!(name: 'Prisoner acknowledgement letter',
                 abbreviation: 'prisoner-acknowledgement',
                 template_type: 'acknowledgement',
                 body: <<~EOF
-                  <p><strong>DATA PROTECTION ACT 2018: SUBJECT ACCESS REQUEST</strong>
+                  <p><%= values.requester_name %><% if values.prison_number.present? %> - <%= values.prison_number %><% if (values.prison_number.gsub(/[,]/, ' ').squeeze(' ').strip).match?(/[ ]/) %> [DELETE AS APPROPRIATE]<% end %><% end %>
+                  <br><%= values.requester_address.gsub("\n", "<br>").html_safe %>
                   <br>
-                  <br>Thank you for your Subject Access Request (SAR) dated <%= 'XXXXXXX' %><%# TODO values.request_dated.strftime('%e %b %Y') %>.
+                  <br><strong>DATA PROTECTION ACT 2018: SUBJECT ACCESS REQUEST</strong>
+                  <br>
+                  <br>Thank you for your Subject Access Request (SAR) dated <%= values.request_dated.strftime('%e %b %Y') %>.
```

Comment:
Request dated is an optional field, so cases without it blow up at this point. 

 we need to use the safe navigation operator here
```suggestion
                  <br>Thank you for your Subject Access Request (SAR) dated <%= values.request_dated&.strftime('%e %b %Y') %>.
```

Reply:
Done